CONTRIBUTING.md

```diff
@@ -1,12 +1,12 @@
 # Contributing to the Windows Community Toolkit Documentation
 
```

Comment:
This one also shouldn't be updated  https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/issues/485